versions.cfg

```diff
@@ -14,8 +14,7 @@ alabaster = 0.7.12
 beautifulsoup4 = 4.11.1
 certifi = 2022.6.15
 cffi = 1.15.0
-# requests 2.28 requires charset-normalizer 2.0.x
-charset-normalizer = 2.0.12
+charset-normalizer = 2.1.0
```

Comment:
Was that fixed in `requests` 2.28.1?

Reply:
Yes, see the commit message of 6f6e7b4a8b17c197d7ba3047e9b0b599c07a0dae:

> Note that requests 2.28.1 (released today) adds compatibility with charset-normalizer 2.1.0.
> See https://github.com/psf/requests/blob/main/HISTORY.md#2281-2022-06-29